finalproject/designreview2.md

```diff
@@ -1,6 +1,10 @@
 # Design Review 2
 
-You will have **8 minutes** to present the following information:
+### This will take place in class on Friday, April 15
+
+Details will be posted here by soon
```

Comment:
Details will be posted here soon
(but it doesn't really matter, we're going to change it anyways)